corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -18,6 +20,16 @@
 class SuiteTest(SimpleTestCase, TestFileMixin):
     file_path = ('data', 'suite')
 
+    def setUp(self):
+        update_toggle_cache(MODULE_FILTER.slug, 'skelly', True, NAMESPACE_DOMAIN)
+        update_toggle_cache(MODULE_FILTER.slug, 'domain', True, NAMESPACE_DOMAIN)
+        update_toggle_cache(MODULE_FILTER.slug, 'example', True, NAMESPACE_DOMAIN)
```

Comment:
I realize this is very fragile, but I'm hoping we can take it out in two weeks. if there's a better way to do this, let me know